src/editorKit.ts

```diff
@@ -120,16 +123,18 @@ export default class EditorKitBase {
        * TODO: If note has changed, release previous temp object URLs.
        */
       let isNewNoteLoad = true
-      if (this.note && this.note.uuid == note.uuid) {
+      if (this.currentNote && this.currentNote.uuid == note.uuid) {
         isNewNoteLoad = false
       }
 
+      this.previousNote = this.currentNote
```

Comment:
```suggestion
      const previousNote = this.currentNote
```

Then you don't need to hang on to previousNote past this block.